Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -679,7 +679,22 @@ def setup_(self, step, stepName, stepDict, k, properties):
     offset = 0.512,
 )
 
-# add here a .513 workflow for GPU vs CPU validation
+upgradeWFs['PatatrackECALOnlyGPUValidation'] = PatatrackWorkflow(
+    digi = {
+        '--accelerators': 'gpu-nvidia',
+        '--procModifiers': 'gpu,gpuValidationEcal'
+    },
+    reco = {
+        '-s': 'RAW2DIGI:RawToDigi_ecalOnly,RECO:reconstruction_ecalOnly,VALIDATION:@ecalOnlyValidation,DQM:@ecalOnly',
+        '--accelerators': 'gpu-nvidia',
+        '--procModifiers': 'gpu,gpuValidationEcal'
```

Comment:
`gpuValidationEcal` could be more generally `gpuValidation`

Reply:
That one would also include the HCAL validation modifier. And maybe others in the future.

Reply:
Yes, that's the idea.
Like the `gpu` modifier can enable the use of GPUs in the various parts of the reconstruction, the `gpuValidation` modifier should enable the GPU validation in all the parts of the reconstruction that run and support it.

Then, since this workflow only runs the ECAL reconstruction and validation, only those two are customised.